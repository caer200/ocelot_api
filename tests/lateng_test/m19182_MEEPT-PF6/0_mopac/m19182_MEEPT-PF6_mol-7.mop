CHARGES

 
H	8.217066 +1 1.969522 +1 21.737409 +1
C	8.598946 +1 1.994076 +1 22.606816 +1
C	9.987794 +1 1.792101 +1 22.763587 +1
C	7.791393 +1 2.225097 +1 23.696534 +1
C	10.494156 +1 1.791426 +1 24.084851 +1
N	10.816886 +1 1.613770 +1 21.653527 +1
H	6.863340 +1 2.378280 +1 23.563325 +1
C	8.308411 +1 2.239958 +1 24.997305 +1
C	9.650706 +1 2.015692 +1 25.189176 +1
S	12.157274 +1 1.609041 +1 24.490056 +1
C	12.183299 +1 1.365861 +1 21.724760 +1
C	10.219101 +1 1.910990 +1 20.313984 +1
H	7.737762 +1 2.402618 +1 25.739106 +1
H	10.009826 +1 2.011625 +1 26.068757 +1
C	12.892586 +1 1.346947 +1 22.951586 +1
C	12.932296 +1 1.127410 +1 20.545148 +1
H	9.539282 +1 2.622500 +1 20.422188 +1
H	10.929757 +1 2.264060 +1 19.722118 +1
C	9.571286 +1 0.722785 +1 19.635674 +1
C	14.282556 +1 1.154430 +1 22.979794 +1
H	12.488047 +1 1.102477 +1 19.705784 +1
C	14.291642 +1 0.932866 +1 20.597535 +1
H	9.051015 +1 1.026220 +1 18.849957 +1
H	8.952199 +1 0.272916 +1 20.263633 +1
O	10.583784 +1 -0.182385 +1 19.224122 +1
H	14.743419 +1 1.163015 +1 23.810387 +1
C	14.979504 +1 0.953130 +1 21.816354 +1
H	14.772876 +1 0.782168 +1 19.792457 +1
C	10.069571 +1 -1.230086 +1 18.390146 +1
H	15.921096 +1 0.829136 +1 21.837140 +1
H	9.603637 +1 -1.903458 +1 18.946469 +1
H	9.418087 +1 -0.856264 +1 17.745175 +1
C	11.204229 +1 -1.878565 +1 17.649624 +1
H	11.739969 +1 -1.186617 +1 17.186618 +1
H	10.846208 +1 -2.500390 +1 16.967643 +1
O	12.027599 +1 -2.591218 +1 18.564476 +1
C	13.119518 +1 -3.195115 +1 17.893683 +1
H	13.632991 +1 -2.507166 +1 17.420624 +1
H	13.697178 +1 -3.642262 +1 18.546910 +1
H	12.783933 +1 -3.852890 +1 17.249730 +1
