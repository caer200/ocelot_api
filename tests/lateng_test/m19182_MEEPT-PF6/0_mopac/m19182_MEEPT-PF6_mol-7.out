 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 175**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon Oct  7 21:31:50 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          8.21706600  *   1.96952200  *  21.73740900  *
     2       C          8.59894600  *   1.99407600  *  22.60681600  *
     3       C          9.98779400  *   1.79210100  *  22.76358700  *
     4       C          7.79139300  *   2.22509700  *  23.69653400  *
     5       C         10.49415600  *   1.79142600  *  24.08485100  *
     6       N         10.81688600  *   1.61377000  *  21.65352700  *
     7       H          6.86334000  *   2.37828000  *  23.56332500  *
     8       C          8.30841100  *   2.23995800  *  24.99730500  *
     9       C          9.65070600  *   2.01569200  *  25.18917600  *
    10       S         12.15727400  *   1.60904100  *  24.49005600  *
    11       C         12.18329900  *   1.36586100  *  21.72476000  *
    12       C         10.21910100  *   1.91099000  *  20.31398400  *
    13       H          7.73776200  *   2.40261800  *  25.73910600  *
    14       H         10.00982600  *   2.01162500  *  26.06875700  *
    15       C         12.89258600  *   1.34694700  *  22.95158600  *
    16       C         12.93229600  *   1.12741000  *  20.54514800  *
    17       H          9.53928200  *   2.62250000  *  20.42218800  *
    18       H         10.92975700  *   2.26406000  *  19.72211800  *
    19       C          9.57128600  *   0.72278500  *  19.63567400  *
    20       C         14.28255600  *   1.15443000  *  22.97979400  *
    21       H         12.48804700  *   1.10247700  *  19.70578400  *
    22       C         14.29164200  *   0.93286600  *  20.59753500  *
    23       H          9.05101500  *   1.02622000  *  18.84995700  *
    24       H          8.95219900  *   0.27291600  *  20.26363300  *
    25       O         10.58378400  *  -0.18238500  *  19.22412200  *
    26       H         14.74341900  *   1.16301500  *  23.81038700  *
    27       C         14.97950400  *   0.95313000  *  21.81635400  *
    28       H         14.77287600  *   0.78216800  *  19.79245700  *
    29       C         10.06957100  *  -1.23008600  *  18.39014600  *
    30       H         15.92109600  *   0.82913600  *  21.83714000  *
    31       H          9.60363700  *  -1.90345800  *  18.94646900  *
    32       H          9.41808700  *  -0.85626400  *  17.74517500  *
    33       C         11.20422900  *  -1.87856500  *  17.64962400  *
    34       H         11.73996900  *  -1.18661700  *  17.18661800  *
    35       H         10.84620800  *  -2.50039000  *  16.96764300  *
    36       O         12.02759900  *  -2.59121800  *  18.56447600  *
    37       C         13.11951800  *  -3.19511500  *  17.89368300  *
    38       H         13.63299100  *  -2.50716600  *  17.42062400  *
    39       H         13.69717800  *  -3.64226200  *  18.54691000  *
    40       H         12.78393300  *  -3.85289000  *  17.24973000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         H          8.2171      1.9695     21.7374
     2         C          8.5989      1.9941     22.6068
     3         C          9.9878      1.7921     22.7636
     4         C          7.7914      2.2251     23.6965
     5         C         10.4942      1.7914     24.0849
     6         N         10.8169      1.6138     21.6535
     7         H          6.8633      2.3783     23.5633
     8         C          8.3084      2.2400     24.9973
     9         C          9.6507      2.0157     25.1892
    10         S         12.1573      1.6090     24.4901
    11         C         12.1833      1.3659     21.7248
    12         C         10.2191      1.9110     20.3140
    13         H          7.7378      2.4026     25.7391
    14         H         10.0098      2.0116     26.0688
    15         C         12.8926      1.3469     22.9516
    16         C         12.9323      1.1274     20.5451
    17         H          9.5393      2.6225     20.4222
    18         H         10.9298      2.2641     19.7221
    19         C          9.5713      0.7228     19.6357
    20         C         14.2826      1.1544     22.9798
    21         H         12.4880      1.1025     19.7058
    22         C         14.2916      0.9329     20.5975
    23         H          9.0510      1.0262     18.8500
    24         H          8.9522      0.2729     20.2636
    25         O         10.5838     -0.1824     19.2241
    26         H         14.7434      1.1630     23.8104
    27         C         14.9795      0.9531     21.8164
    28         H         14.7729      0.7822     19.7925
    29         C         10.0696     -1.2301     18.3901
    30         H         15.9211      0.8291     21.8371
    31         H          9.6036     -1.9035     18.9465
    32         H          9.4181     -0.8563     17.7452
    33         C         11.2042     -1.8786     17.6496
    34         H         11.7400     -1.1866     17.1866
    35         H         10.8462     -2.5004     16.9676
    36         O         12.0276     -2.5912     18.5645
    37         C         13.1195     -3.1951     17.8937
    38         H         13.6330     -2.5072     17.4206
    39         H         13.6972     -3.6423     18.5469
    40         H         12.7839     -3.8529     17.2497

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C17 H19 N O2 S  =    40 atoms



      MOLECULAR POINT GROUP   :   C1  

          COMPUTED CHARGE ON SYSTEM: +0

   COMPUTED CHARGE ON SYSTEM = +0. THIS AGREES WITH THE CHARGE IN THE DATA-SET.

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
